internal/cmd/profile/get_alias_with_profile.go

```diff
@@ -0,0 +1,40 @@
+package profile
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spacelift-io/spacectl/client/session"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	apiTokenProfile *session.Profile
+)
+
+func getAliasWithAPITokenProfile(cliCtx *cli.Context) error {
+	if err := getAlias(cliCtx); err == nil {
+		return nil
+	}
+
+	userHomeDir, err := os.UserHomeDir()
+	if err != nil {
+		return fmt.Errorf("could not find user home directory: %w", err)
+	}
+
+	manager, err := session.NewProfileManager(filepath.Join(userHomeDir, session.SpaceliftConfigDirectory))
+	if err != nil {
+		return fmt.Errorf("could not create profile manager: %w", err)
+	}
+
+	profile := manager.Current()
+	if profile != nil && profile.Credentials.Type == session.CredentialsTypeAPIToken {
+		apiTokenProfile = profile
+	} else {
+		return errors.New("command is supported only for exisiting API Token profile. Please use `spacectl profile login <alias>` instead")
```

Comment:
```suggestion
		return errors.New("command is only supported when using an existing API Token profile. Please use `spacectl profile login <alias>` instead")
```

There was a typo in `exisiting`, but I've made a suggestion about rewording slightly. Mainly `s/supported only/only supported` - both are valid, but IMO "only supported" reads more naturally.